transports/transport-highway/src/main/java/io/servicecomb/transport/highway/HighwayClient.java

```diff
@@ -37,78 +40,79 @@
-    TcpClientConfig tcpClientConfig = new TcpClientConfig();
-    tcpClientConfig.setRequestTimeoutMillis(AbstractTransport.getRequestTimeout());
-
-    if (this.sslEnabled) {
-      SSLOptionFactory factory =
-          SSLOptionFactory.createSSLOptionFactory(SSL_KEY, null);
-      SSLOption sslOption;
-      if (factory == null) {
-        sslOption = SSLOption.buildFromYaml(SSL_KEY);
-      } else {
-        sslOption = factory.createSSLOption();
-      }
-      SSLCustom sslCustom = SSLCustom.createSSLCustom(sslOption.getSslCustomClass());
-      VertxTLSBuilder.buildClientOptionsBase(sslOption, sslCustom, tcpClientConfig);
-    }
-    return tcpClientConfig;
-  }
-
-  public void send(Invocation invocation, AsyncResponse asyncResp) throws Exception {
-    HighwayClientConnectionPool tcpClientPool = clientMgr.findThreadBindClientPool();
-
-    OperationMeta operationMeta = invocation.getOperationMeta();
-    OperationProtobuf operationProtobuf = ProtobufManager.getOrCreateOperation(operationMeta);
-
-    HighwayClientConnection tcpClient = tcpClientPool.findOrCreateClient(invocation.getEndpoint().getEndpoint());
-    HighwayClientPackage clientPackage = new HighwayClientPackage(invocation, operationProtobuf, tcpClient);
-    log.debug("Calling method {} of {} by highway", operationMeta.getMethod(), invocation.getMicroserviceName());
-    tcpClientPool.send(tcpClient, clientPackage, ar -> {
-      // 此时是在网络线程中，转换线程
-      invocation.getResponseExecutor().execute(() -> {
-        if (ar.failed()) {
-          // 只会是本地异常
-          asyncResp.consumerFail(ar.cause());
-          return;
-        }
-
-        // 处理应答
-        try {
-          Response response =
-              HighwayCodec.decodeResponse(invocation,
-                  operationProtobuf,
-                  ar.result(),
-                  tcpClient.getProtobufFeature());
-          asyncResp.complete(response);
-        } catch (Throwable e) {
-          asyncResp.consumerFail(e);
-        }
-      });
-    });
-  }
+	private static final Logger log = LoggerFactory.getLogger(HighwayClient.class);
+
+	private static final String SSL_KEY = "highway.consumer";
+
+	private ClientPoolManager<HighwayClientConnectionPool> clientMgr = new ClientPoolManager<>();
+
+	private final boolean sslEnabled;
+
+	public HighwayClient(boolean sslEnabled) {
+		this.sslEnabled = sslEnabled;
+	}
+
+	public void init(Vertx vertx) throws Exception {
+		TcpClientConfig config = createTcpClientConfig();
+
+		DeploymentOptions deployOptions = VertxUtils.createClientDeployOptions(clientMgr,
+				HighwayConfig.getClientThreadCount(), HighwayConfig.getClientConnectionPoolPerThread(), config);
+
+		VertxUtils.blockDeploy(vertx, HighwayClientVerticle.class, deployOptions);
+	}
+
+	private TcpClientConfig createTcpClientConfig() {
+		TcpClientConfig tcpClientConfig = new TcpClientConfig();
+		DynamicLongProperty prop = DynamicPropertyFactory.getInstance().getLongProperty("cse.request.timeout", 30000);
```

Comment:
"cse.request.timeout"/30000 please use const
maybe AbstractTransport add a new method to getRequestTimeoutProperty is better